.gitignore

```diff
@@ -4,7 +4,6 @@ coverage
 dist
 node_modules
 npm-debug.log
-package-lock.json
```

Comment:
It is Apollo convention to exclude lock files.

Reply:
Do you know why that's the case?  Interested in learning more.

Anyways I am willing to drop this though.

Reply:
With common use of npm and yarn it can lead to odd issues unless you are aware of the lock files. Additionally with semver there is no reason not to pickup bug fixes.

Reply:
I see.  I've dropped the commit from my branch.